src/disco/containers/DiscoPane.js

```diff
@@ -176,8 +185,9 @@ export function loadedAddons(state) {
   );
 }
 
-export function mapStateToProps(state) {
+export function mapStateToProps(state, ownProps) {
   return {
+    platform: ownProps.params.platform,
```

Comment:
Yeah sorry, actually you should not do that. Remove the `ownProps` arg here, and use `params.platform` in the component.